packages/fxa-auth-server/CHANGELOG.md

```diff
@@ -1,9 +1,6 @@
 ## 1.136.2
 
-### Bug fixes
-
-* backendService: Update test to ensure we don't send a body in GET requests. (5f1c5dfab)
```

Comment:
how come this disappeared?

Reply:
Oh, this was a duplicated I accidentally introduce in previous point release.

Reply:
Oh ok! got it! 👍